documentation/cloud-manager-guide/admin/alert-management/alert-management-overview.md

```diff
@@ -5,4 +5,143 @@ description: Alert management
 ---
 # Alert Management
 
-**This page is a WIP - to be included with beta release**
+Custom alerts can be created in Environment Operations Center to help your teams monitor the health and operations of your environments. Once created, alerts are sent to your specified communication channels to keep you up to date on important changes, potential issues, or errors. This guide outlines the steps to create and manage alerts in Environment Operations Center.
+
+[!note] An integration channel to receive the alert must be created prior to setting up the alert. For details on adding communication channel integrations, see the [integrations](../integrations/manage-integrations.md) guide.
+
+## Getting started
+
+Navigate to the *Alerts* tab in the *Admin* section of environment operations center. From this tab, you can add new environment monitoring alerts and manage existing alerts.
+
+![image description](images/alerts-tab.png)
+
+All existing alerts are listed on the *Alerts* tab including the alert name, notification channel, when it was last updated, and the severity of the alert.
+
+![image description](images/alert-details.png)
```

Comment:
Can combine screenshots on line 16 & 20 and move the text on line 18 up to line 14, preventing the number of single lines of text, making it a bit easier to follow.